### PR TITLE
docs: update README for application metadata

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**
As I had a hard time finding the right metadata, let's make sure others don't go through the same ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,21 @@ Follow these steps to obtain them:
 
 See more details on how to create a new Azure AD application account in your organizaiton's directory and add it to your Partner Center account [here](https://docs.microsoft.com/en-gb/windows/uwp/publish/add-users-groups-and-azure-ad-applications#create-a-new-azure-ad-application-account-in-your-organizations-directory-and-add-it-to-your-partner-center-account).
 
+## Obtaining App Metadata
+
+### Seller ID
+
+The Seller ID can be found by clicking the gear icon in the upper right corner of the [Partner Center](https://partner.microsoft.com) and selecting `Account Settings`. You can find the Seller ID under [Legal info](https://partner.microsoft.com/en-us/dashboard/account/v3/organization/legalinfo)
```
Comment (azchohfi) [inline README.md:46]:
```suggestion
The Seller ID can be found by clicking the gear icon in the upper right corner of the [Partner Center](https://partner.microsoft.com) and selecting `Account Settings`. You can find the Seller ID under [Legal info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo)
```

Comment (azchohfi) [reply README.md:46]: The links shouldn't have the en-us bit. It automatically redirects to the right language/locale.

**Review by azchohfi — CHANGES_REQUESTED** [review]

**Comment by azchohfi** [comment]
Thank you for your contribution! I'll send another PR fixing the wrong en-us/en-gb we have in our readme file, but lets make sure we don't add any more of them.

File: README.md (hunk 1 above)
Comment (JanDeDobbeleer) [reply README.md:46]: Absolutely right, I'll adjust the original commit in a few.

**Review by azchohfi — APPROVED** [review]

**Pull request merged**